fraud_detection/testing.py

```diff
@@ -20,6 +20,10 @@
 import strawberryfields as sf
 from strawberryfields.ops import Dgate, BSgate, Kgate, Sgate, Rgate
 
+import sys
+sys.path.append("..")
+import version_check
```

Comment:
How come you need to append to the path here? Does `import ..version_check` not work?

Reply:
Yes I just tried again and couldn't get it to work - I think it's the lack of an `__init__.py`